CMakeLists.txt

```diff
@@ -289,38 +292,38 @@ add_subdirectory(src/dyad)
 #-----------------------------------------------------------------------------
 # Configure the config.cmake file for the build directory
 #-----------------------------------------------------------------------------
-configure_file(
-        ${CMAKE_CURRENT_SOURCE_DIR}/CMake/${PROJECT_NAME}-config.cmake.build.in
-        ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cmake/${PROJECT_NAME}/${PROJECT_NAME}-config.cmake @ONLY
-)
-
-configure_file(
-        ${CMAKE_CURRENT_SOURCE_DIR}/CMake/${PROJECT_NAME}-config.cmake.install.in
-        ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cmake/${PROJECT_NAME}/install/${PROJECT_NAME}-config.cmake @ONLY
-)
-install(
-        FILES
-        ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cmake/${PROJECT_NAME}/install/${PROJECT_NAME}-config.cmake
-        DESTINATION
-        ${DYAD_LIBDIR}/cmake/${PROJECT_NAME}
-)
+# configure_file(
```

Comment:
Why do we have these commented. We should remove them if we are using the `configure_package_config_file`

Reply:
I didn't want to delete anything until I finish final testing. Once I finish my current round of testing, I'll remove them.

Reply:
Removed